numpy/lib/histograms.py

```diff
@@ -436,7 +447,7 @@ def histogram_bin_edges(a, bins=10, range=None, weights=None):
     below, :math:`h` is the binwidth and :math:`n_h` is the number of
     bins. All estimators that compute bin counts are recast to bin width
     using the `ptp` of the data. The final bin count is obtained from
-    ``np.round(np.ceil(range / h))`.
+    `np.round(np.ceil(range / h))`.
```

Comment:
I think the correct version is to have two ticks on both sides. That's the sphinx symbol for a code block, vs the single tick which is "format how you see fit".

Reply:
Oh dear, this entire docstring uses primarily single backticks

Reply:
Yeah you're right, the entire docstring primarily uses single backticks though.